src/app/integrations/edit-page/step-configure/step-configure.component.ts

```diff
@@ -31,29 +31,27 @@ export class IntegrationsStepConfigureComponent extends FlowPage
   formConfig: any;
   cfg: any = undefined;
 
-  constructor(
-    public currentFlow: CurrentFlow,
-    public route: ActivatedRoute,
-    public router: Router,
-    public formFactory: FormFactoryService,
-    public formService: DynamicFormService,
-    public detector: ChangeDetectorRef,
-    public stepStore: StepStore,
-  ) {
```

Comment:
This change and others in this file makes me think you might still need to get [prettier](https://github.com/prettier/prettier) going in your IDE